### PR TITLE
Authentication text

### LINKED ISSUE #387: Change `ERROR    No saved authentication token found!` when `dds auth info` run
Tester:
```
Without being logged in “dds auth info” shows the message: 
ERROR    No saved authentication token found!

Suggestion: instead of “ERROR …” it could perhaps be something like: 
“INFO: No sessions found. Please authenticate yourself and start a new session!”
Or even better, use the message from “dds ls” without a session:
INFO     No saved token found, or token has expired, proceeding with authentication
```

We can look into changing the `ERROR` to `INFO` and changing the message to `No sessions found. Please authenticate yourself and start a new session`. Will not automatically start authentication in this case though.

### PR DESCRIPTION
**Pull request by zishanmirza**
This patch updates the authentication text and switches "LOG.error" to "LOG.info".

Fixes #387.

Before submitting a PR to the `dev` branch:
- [X] Tests passing
- [X] Black formatting
- [X] Rebase/merge the `dev` branch
- [ - ] Note in the CHANGELOG

### CODE REVIEW

File: dds_cli/auth.py [hunk 1]
```diff
@@ -40,7 +40,9 @@ def check(self):
             token = token_file.read_token()
             token_file.token_report(token=token)
         else:
-            LOG.error("[red]No saved authentication token found![/red]")
+            LOG.info(
+                "[red]No saved token found, or token has expired, proceeding with authentication![/red]"
```
Comment (inaod568) [inline dds_cli/auth.py:44]: For `dds auth info` it doesn't proceed with the authentication, I suggest this instead: 

```suggestion
                "[red]No saved token found, or token has expired. Authenticate yourself with `dds auth login` to use this functionality.[/red]"
```

Comment (zishanmirza) [reply dds_cli/auth.py]: Sure, I have updated the text.

File: dds_cli/auth.py [hunk 2]
```diff
@@ -40,7 +40,9 @@ def check(self):
             token = token_file.read_token()
             token_file.token_report(token=token)
         else:
-            LOG.error("[red]No saved authentication token found![/red]")
+            LOG.info(
+                "[red]No saved token found, or token has expired, Authenticate yourself with `dds auth login` to use this functionality.![/red]"
```
Comment (inaod568) [inline dds_cli/auth.py:44]: Sorry. Just a little comma that should be a dot. :)  
```suggestion
                "[red]No saved token found, or token has expired. Authenticate yourself with `dds auth login` to use this functionality.![/red]"
```

**Review by inaod568 — CHANGES_REQUESTED** [review]
Tiiny change.

File: dds_cli/auth.py (hunk 2 above)
Comment (zishanmirza) [reply dds_cli/auth.py]: Sure, I have updated the text.

**Pull request merged**